### PR TITLE
[chore] Clarify that tail_sampling is not a sampling processor for ordering

### PR DESCRIPTION
**Pull request by gebn**
**Description:** The advice that sampling processors be placed before context-sensitive processors does not apply to tail_sampling. This PR will hopefully prevent others from falling into the same trap we did.

**Link to tracking Issue:** #8064

### CODE REVIEW

**Review by atoulme — APPROVED** [review]

**Comment by gebn** [comment]
Thanks @djaglowski, just waiting for corporate sign-off on the CLA.

**Comment by gebn** [comment]
@djaglowski Sorted now!

**Review by crobert-1 — APPROVED** [review]

**Review by atoulme — APPROVED** [review]

File: processor/README.md
Comment (bogdandrutu) [inline processor/README.md:33]: Not sure I understand this, can you explain to me?

Comment (gebn) [reply processor/README.md:33]: `tail_sampling` is the name of a sampling processor, however if it is added at this point, it will break `k8sattributes`. This line is saying "Add sampling processors, apart from `tail_sampling`". `tail_sampling` needs to be added below. It sounds like it may need a reword - how can that be made clearer?

**Comment by gebn** [comment]
@bogdandrutu PTAL

**Pull request closed without merging**
